tests/components/homeassistant/common.yaml

```
@@ -32,6 +32,11 @@ wifi:
 
 api:
 
+switch:
+  - platform: homeassistant
+    entity_id: switch.my_cool_switch
```

To make sure everyone understand directly that the entity_id is related to home-assistant would it not be to change:  
```suggestion
    homeassistant_entity: switch.my_cool_switch
```

---

I see that the depending PR is already merged.

---

If we changed it to something else, it would be inconsistent with the existing sensor/binary_sensor/text_sensor implementations.

---

Yeah, when you decide to change it then those should change as well.
As i said i'm to late for this request :(